e2e/menuWorkExperience.spec.ts

```diff
@@ -99,8 +99,8 @@ test.describe('Work experience menu input', () => {
 
       await fillAndVerify(page, 'work_0_position', 'Position');
       await fillAndVerify(page, 'work_0_name', 'Name');
-      await fillAndVerify(page, 'work_0_startDate', '2000-01', 'January 2000');
-      await fillAndVerify(page, 'work_0_endDate', '2000-12', 'December 2000');
+      await fillAndVerify(page, 'work_0_startDate', '2000-01', '2000');
+      await fillAndVerify(page, 'work_0_endDate', '2000-12', '2000');
```

Comment:
I should update this function to pass the decoration function into this function so it's more resilient